components/style/properties/data.py

```diff
@@ -220,12 +230,19 @@ def new_style_struct(self, *args, **kwargs):
     def active_style_structs(self):
         return [s for s in self.style_structs if s.additional_methods or s.longhands]
 
+    def add_prefixed_aliases(self, property_data):
+        # FIXME Servo's DOM architecture doesn't support vendor-prefixed properties.
```

Comment:
FWIW, Gecko prepends `_` to property names in WebIDL files. See [CSS2Properties.webidl](https://dxr.mozilla.org/mozilla-central/source/obj-x86_64-pc-linux-gnu/dom/bindings/CSS2Properties.webidl). There is probably some magic in the codegen to handle this.